src/bin/html5-parser-test.rs

```diff
@@ -1,28 +1,40 @@
-use gosub_engine::testing::tree_construction::fixture_from_filename;
-use gosub_engine::testing::FIXTURE_ROOT;
+use gosub_engine::testing::tree_construction::fixture::{
+    get_fixture_root_path, read_fixture_from_path,
+};
+use gosub_engine::testing::tree_construction::Harness;
 use gosub_engine::types::Result;
 use std::io::Write;
-use std::path::Path;
+use std::path::{Path, PathBuf};
 use walkdir::WalkDir;
 
 fn main() -> Result<()> {
-    let mut files = get_files_from_path(format!("{}/tree-construction", FIXTURE_ROOT).as_str());
+    let mut files = get_files_from_path(get_fixture_root_path());
```

Comment:
All fixture loading / tree stuff is moved to fixture.rs

Reply:
I'd pitch for not using the `get_` prefix.  This feels foreign to Rust.

Reply:
I'm juggling three different languages in my daily work. It's hard to keep all the idioms straight :-)   I'll remove the ones in the test suite but i reckon we would need to do this at more places later on